.github/ISSUE_TEMPLATE/01-bug-report.yml

```diff
@@ -0,0 +1,80 @@
+name: Bug Report
+description: File a bug report
+title: "[BUG]: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this bug report.
+        Please fill out the following information to the best of your ability to help us resolve the issue.
+  - type: dropdown
+    id: product
+    attributes:
+      label: Product
+      description: "Which product does this bug report apply to?"
+      options:
+        - "Touch Adaptation Kit Editor (TAK Editor)"
+        - "Touch Adaptation Kit Command Line Tool (TAK CLI)"
+        - "Touch Adaptation Kit Schemas (TAK Schemas)"
+        - "Other (please specify in the Reproduction Steps section)"
+      multiple: true
+    validations:
+      required: true
+  - type: dropdown
+    id: os
+    attributes:
+      label: "Operating System"
+      description: "Which operating system are you using?"
+      options:
+        - "Windows"
+        - "macOS"
+        - "Linux"
+      multiple: false
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: "Version"
+      description: "What version of the product are you using? If using multiple products, please list all versions."
```

Comment:
Should we provide brief instruction on how to get versions here? It would be convenient for reporter to grab the command and copy the result to fill in this question.

Reply:
Done! :)